samples/openapi3/client/petstore/python-experimental/testfiles/ecP521.pem

```diff
@@ -0,0 +1,10 @@
+-----BEGIN ENCRYPTED PRIVATE KEY-----
+MIIBTzBJBgkqhkiG9w0BBQ0wPDAbBgkqhkiG9w0BBQwwDgQIl5fhsgDB0BICAgPo
+MB0GCWCGSAFlAwQBAgQQEprktYMTqB+wmvDAipQDigSCAQACy7C5dcYGK7d+fvm9
+QbjAfdNkqb1lTk21ni53ApJdY2sHF9+Aw/bz7DmuJVx5KS0hRPX8F3dy1u0V+DIA
+3dxjlj6zmUmh/emLgRWz84cWYBzVCt80B5H29gx4ti6JQeXZfc4TXUVApLov0WX0
+D9ShtEjzg14SEqtQsTgmA5jqkzAYNkGbEsWr0yyr/eqArTbrJblXWNhlWk1AodOz
+JLXyypDx+Df10updwHfntWegu3UY8W+3cVjSi2K9QAwyMe2RKzEcsCFrd3OiH5ys
+SrDExnhsdYLH5x4mdrw4+DSqMT4r7/J8hLHxjMPPTr6nGbd3pU2Q0ZN6CMfLd77S
+IQuN
+-----END ENCRYPTED PRIVATE KEY-----
```

Comment:
@sebastien-rosset why are these new?

Reply:
This file is generated during the unit tests, this is done to test the HTTP signature scheme. Do you see circleCI complaining the file needs to be committed? Or are you asking about the purpose of the file?

Reply:
Ah, I see that here:
https://github.com/OpenAPITools/openapi-generator/blob/4f350bc01cb3aaadf71122ee9d4f1b975e1f2295/samples/openapi3/client/petstore/python-experimental/tests/test_http_signature.py#L222

I was asking about the purpose of the file when my PR didn't appear to generate.
The tests generate it.

Do you mind if I add a [tearDown](https://docs.python.org/2/library/unittest.html#unittest.TestCase.tearDown) method to clean up these files?
If we leave the code as-is then they can show up unexpectedly in future PRs.

Reply:
> Ah, I see that here:
> https://github.com/OpenAPITools/openapi-generator/blob/4f350bc01cb3aaadf71122ee9d4f1b975e1f2295/samples/openapi3/client/petstore/python-experimental/tests/test_http_signature.py#L222
> 
> I was asking about the purpose of the file when my PR didn't appear to generate it.
> The tests generate it.
> 
> Do you mind if I add a [tearDown](https://docs.python.org/2/library/unittest.html#unittest.TestCase.tearDown) method to remove these pem files?

sure, that would be great.

> If we leave the code as-is then they can show up unexpectedly in future PRs.
> Also, it looks like we may want to change setUp to setUpClass so that we run set up only once per class rather than once before every method test as we are currently doing.

Agreed.

Reply:
These files are now cleaned up in the tearDownClass method